File: backend/config/routes/workspace.py

from app.api import all_paths
from app.api import config
from app.api.resources import resources_api
   
def set_routes(app):
    app.add_url_rule("/api/allPaths", "api_all_paths",
                     all_paths.api_all_paths)
    app.add_url_rule("/api/saveConfig", "api_save_config",
                     config.api_save_config, methods=["POST"])
    app.add_url_rule("/api/loadConfig", "api_load_config",
                     config.api_load_config)
    
    #registro usando Blueprints
    app.register_blueprint(resources_api)

File: frontend/src/api/workspace.js
import { axiosApi } from './axios'

const paths = {
  allPaths: '/allPaths',
  getImg: '/image/load',
  saveConfig: '/saveConfig',
  loadConfig: '/loadConfig',
  getResource: '/api/resources/'
}

export default {
  allPaths: async (data) => axiosApi.get(paths.allPaths, {
    params: data
  }),
  getImg: async (data) => axiosApi.get(paths.getImg, {
    params: data
  }),
  saveConfig: async (data) => axiosApi.post(paths.saveConfig, data),
  loadConfig: async () => axiosApi.get(paths.loadConfig),
  getResource: async (data) => axiosApi.get(paths.getResource, {
    params: data
  })
}
